src/main/java/domain/participant/Result.java

```diff
@@ -1,17 +1,32 @@
 package domain.participant;
 
 public enum Result {
-    WIN,
-    LOSE,
-    DRAW;
+    BLACKJACK(1.5),
+    WIN(1),
+    DRAW(0),
+    LOSE(-1);
 
-    public Result reverseResult() {
-        if (this == WIN) {
+    private final double profitRate;
+
+    Result(double profitRate) {
+        this.profitRate = profitRate;
+    }
```

Comment:
```suggestion
    PlayerResult(double profitRate) {
        this.profitRate = profitRate;
    }
```
로직을 확인해보니 이런 이름이 더 어울리는 것 같아요~

Reply:
수정해보겠습니다!